.pre-commit-config.yaml

```diff
@@ -29,6 +24,12 @@ repos:
       rev: 5.12.0
       hooks:
           - id: isort
+    - repo: https://github.com/pre-commit/mirrors-mypy
+      rev: v1.0.1
+      hooks:
+          - id: mypy
+            additional_dependencies: [numpy==1.22.0, types-requests]
+            exclude: tests/|docs/|temp/|spatialdata/_core/reader.py
```

Comment:
is this the altest version? I would just run `pre-commit autoupdate` to automatically update all versions of pre-commit (it might be, just to be sure)

Reply:
![image](https://user-images.githubusercontent.com/12818242/224980884-125b93de-268a-4070-8f53-3149a4aaf612.png)